prober/icmp.go

```diff
@@ -80,6 +80,11 @@ func ProbeICMP(ctx context.Context, target string, module config.Module, registr
 	}
 
 	registry.MustRegister(durationGaugeVec)
+	ttlGauge := prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "probe_icmp_reply_ttl_total",
+		Help: "Number of router hops returning from the target",
```

Comment:
No it's not, it's the TTL of the received packet - these are very different things.

Reply:
Now 'Replied packet hop limit (TTL for ipv4)'